packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -81,12 +77,6 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
         // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we try removing these eslint comments and fix the component by filling in the dependency array?

Reply:
I think If we add `getApiToken()` to dependency array here, it will cause an infinite loop. Since, `getApiToken()` is an async function which also sets state...everytime it will change state (after fetching response) it triggers re-render and useEffect will run again, create function again thus, leading an infinite loop.